examples/REEL/backend/data/movies_by_genre.json

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
This file is used by the /onboarding route because we just hard-coded some popular movies that will be suggested since we don't have an sort of popularity metric in the database.